### ISSUE TITLE
Issue building linaria with typescript/rollup

### ISSUE THREAD

**Issue by fxOne** [callstack/linaria#698]
## Environment

- Linaria version: 2.0.1
- Bundler (+ version): rollup 2.33.1
- Node.js version: 12.13.0
- OS: WSL2 with Ubuntu 20.04

## Description

On building the package `draft-js-emoji-plugin` we are getting the following error since the change to typescript:

```
yarn run v1.18.0
$ ../node_modules/.bin/rollup --config ../rollup.config.ts.js

./src/index.tsx → ./lib/index.cjs.js...
[!] (plugin babel) Error: /home/hcomuser/dev/draft-js-plugins/draft-js-emoji-plugin/src/theme.ts: An unexpected runtime error ocurred during dependencies evaluation:
SyntaxError: /home/hcomuser/dev/draft-js-plugins/draft-js-emoji-plugin/src/theme.ts: Support for the experimental syntax 'flow' isn't currently enabled (2:8):

  1 | import { css } from 'linaria';
> 2 | export interface EmojiPluginTheme {
    |        ^
  3 |   emoji?: string;
  4 |   emojiSuggestions?: string;
  5 |   emojiSuggestionsEntry?: string;

Add @babel/preset-flow (https://git.io/JfeDn) to the 'presets' section of your Babel config to enable transformation.
If you want to leave it as-is, add @babel/plugin-syntax-flow (https://git.io/vb4yb) to the 'plugins' section to enable parsing.
    at Parser._raise (/home/hcomuser/dev/draft-js-plugins/node_modules/@babel/parser/src/parser/error.js:60:45)
    at Parser.raiseWithData (/home/hcomuser/dev/draft-js-plugins/node_modules/@babel/parser/src/parser/error.js:55:17)
    at Parser.expectOnePlugin (/home/hcomuser/dev/draft-js-plugins/node_modules/@babel/parser/src/parser/util.js:157:18)
    at Parser.isExportDefaultSpecifier (/home/hcomuser/dev/draft-js-plugins/node_modules/@babel/parser/src/parser/statement.js:1845:16)
    at Parser.maybeParseExportDefaultSpecifier (/home/hcomuser/dev/draft-js-plugins/node_modules/@babel/parser/src/parser/statement.js:1722:14)
    at Parser.parseExport (/home/hcomuser/dev/draft-js-plugins/node_modules/@babel/parser/src/parser/statement.js:1667:29)
    at Parser.parseStatementContent (/home/hcomuser/dev/draft-js-plugins/node_modules/@babel/parser/src/parser/statement.js:249:25)
    at Parser.parseStatement (/home/hcomuser/dev/draft-js-plugins/node_modules/@babel/parser/src/parser/statement.js:151:17)
    at Parser.parseBlockOrModuleBlockBody (/home/hcomuser/dev/draft-js-plugins/node_modules/@babel/parser/src/parser/statement.js:871:25)
    at Parser.parseBlockBody (/home/hcomuser/dev/draft-js-plugins/node_modules/@babel/parser/src/parser/statement.js:841:10)

It may happen when your code or third party module is invalid or uses identifiers not available in Node environment, eg. window.
Note that line numbers in above stack trace will most likely not match, because Linaria needed to transform your code a bit.

src/theme.ts
Error: /home/hcomuser/dev/draft-js-plugins/draft-js-emoji-plugin/src/theme.ts: An unexpected runtime error ocurred during dependencies evaluation:
SyntaxError: /home/hcomuser/dev/draft-js-plugins/draft-js-emoji-plugin/src/theme.ts: Support for the experimental syntax 'flow' isn't currently enabled (2:8):

  1 | import { css } from 'linaria';
> 2 | export interface EmojiPluginTheme {
    |        ^
  3 |   emoji?: string;
  4 |   emojiSuggestions?: string;
  5 |   emojiSuggestionsEntry?: string;

Add @babel/preset-flow (https://git.io/JfeDn) to the 'presets' section of your Babel config to enable transformation.
If you want to leave it as-is, add @babel/plugin-syntax-flow (https://git.io/vb4yb) to the 'plugins' section to enable parsing.
    at Parser._raise (/home/hcomuser/dev/draft-js-plugins/node_modules/@babel/parser/src/parser/error.js:60:45)
    at Parser.raiseWithData (/home/hcomuser/dev/draft-js-plugins/node_modules/@babel/parser/src/parser/error.js:55:17)
    at Parser.expectOnePlugin (/home/hcomuser/dev/draft-js-plugins/node_modules/@babel/parser/src/parser/util.js:157:18)
    at Parser.isExportDefaultSpecifier (/home/hcomuser/dev/draft-js-plugins/node_modules/@babel/parser/src/parser/statement.js:1845:16)
    at Parser.maybeParseExportDefaultSpecifier (/home/hcomuser/dev/draft-js-plugins/node_modules/@babel/parser/src/parser/statement.js:1722:14)
    at Parser.parseExport (/home/hcomuser/dev/draft-js-plugins/node_modules/@babel/parser/src/parser/statement.js:1667:29)
    at Parser.parseStatementContent (/home/hcomuser/dev/draft-js-plugins/node_modules/@babel/parser/src/parser/statement.js:249:25)
    at Parser.parseStatement (/home/hcomuser/dev/draft-js-plugins/node_modules/@babel/parser/src/parser/statement.js:151:17)
    at Parser.parseBlockOrModuleBlockBody (/home/hcomuser/dev/draft-js-plugins/node_modules/@babel/parser/src/parser/statement.js:871:25)
    at Parser.parseBlockBody (/home/hcomuser/dev/draft-js-plugins/node_modules/@babel/parser/src/parser/statement.js:841:10)

It may happen when your code or third party module is invalid or uses identifiers not available in Node environment, eg. window.
Note that line numbers in above stack trace will most likely not match, because Linaria needed to transform your code a bit.

    at PluginPass.enter (/home/hcomuser/dev/draft-js-plugins/node_modules/linaria/src/babel/extract.ts:188:21)
    at newFn (/home/hcomuser/dev/draft-js-plugins/node_modules/@babel/traverse/lib/visitors.js:175:21)
    at NodePath._call (/home/hcomuser/dev/draft-js-plugins/node_modules/@babel/traverse/lib/path/context.js:55:20)
    at NodePath.call (/home/hcomuser/dev/draft-js-plugins/node_modules/@babel/traverse/lib/path/context.js:42:17)
    at NodePath.visit (/home/hcomuser/dev/draft-js-plugins/node_modules/@babel/traverse/lib/path/context.js:92:31)
    at TraversalContext.visitQueue (/home/hcomuser/dev/draft-js-plugins/node_modules/@babel/traverse/lib/context.js:112:16)
    at TraversalContext.visitSingle (/home/hcomuser/dev/draft-js-plugins/node_modules/@babel/traverse/lib/context.js:84:19)
    at TraversalContext.visit (/home/hcomuser/dev/draft-js-plugins/node_modules/@babel/traverse/lib/context.js:140:19)
    at Function.traverse.node (/home/hcomuser/dev/draft-js-plugins/node_modules/@babel/traverse/lib/index.js:82:17)
    at traverse (/home/hcomuser/dev/draft-js-plugins/node_modules/@babel/traverse/lib/index.js:64:12)
```
If I remove the [`.${emojiSelectPopoverScrollbar}`](https://github.com/draft-js-plugins/draft-js-plugins/blob/2e1e49f2e1874bd88d73f0d151e19b251ff57913/draft-js-emoji-plugin/src/theme.ts#L242) like syntax the build works fine. Also converting the theme.ts file back to .js/.d.ts solves the issue too.

A similar issue can be found here: callstack/linaria#403

## Reproducible Demo

- checkout https://github.com/draft-js-plugins/draft-js-plugins.git
- `yarn install`
- `cd draft-js-emoji-plugin`
- `yarn build:js`

**Comment by Anber** [comment]
Hi @fxOne
Thank you for your report!

The problem here appears because Linaria doesn't see a Babel config. Our Webpack loader knows how to work with Yarn and Lerna monorepos, but Rollup loader is a bit dumber. I'll try to fix it ASAP.

**Comment by Anber** [comment]
Should be fixed in `2.0.2`.

By the way, your `scripts/build-css.js` tries to resolve `.js` files, whereas all sources in `ts` and `tsx`.
